modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+
+[id='odo-sample-applications']
+= Sample applications
+include::openshift-developer-cli-attributes.adoc[]
+
+:context: odo-sample-applications
+
+{odo-title} is compatible with any language or runtime listed within the OpenShift catalog of component types. For example:
+
+----
```

Comment:
This is new output from `odo catalog list components` it shows the supported and unsupported components, we should probably use this.
```
Odo Supported OpenShift Components:
NAME       PROJECT       TAGS
java       openshift     11,8,latest
nodejs     openshift     10,8,8-RHOAR,latest

Odo Unsupported OpenShift Components:
NAME              PROJECT       TAGS
dotnet            openshift     2.1,2.2,latest
golang            openshift     1.11.5,latest
httpd             openshift     2.4,latest
modern-webapp     openshift     10.x,latest
nginx             openshift     1.10,1.12,latest
perl              openshift     5.24,5.26,latest
php               openshift     7.0,7.1,7.2,latest
python            openshift     2.7,3.6,latest
ruby              openshift     2.4,2.5,latest
```